pkg/storage/export/export/export.go

```diff
@@ -599,6 +609,65 @@ func (ctrl *VMExportController) createCertSecretManifest(vmExport *exportv1.Virt
 	}, nil
 }
 
+// handleVMExportToken checks if a secret has been specified for the current export object and, if not, creates one specific to it
+func (ctrl *VMExportController) handleVMExportToken(vmExport *exportv1.VirtualMachineExport) error {
+	if vmExport.Status == nil {
+		vmExport.Status = &exportv1.VirtualMachineExportStatus{
+			Phase: exportv1.Pending,
```

Comment:
Since we are always setting this here, do we still need to set Pending and the conditions in `updateCommonVMExportStatusFields`?

Reply:
I guess we don't, however, I like how we check before dereferencing it. It shouldn't be necessary anyway,  so I can remove the one in `updateCommonVMExportStatusFields`.